lib/myxql/client.ex

```diff
@@ -142,6 +152,19 @@ defmodule MyXQL.Client do
     :ok
   end
 
+  def send_com(client, coms) when is_list(coms) do
+    sequence_num = 0
```

Comment:
Is it correct for sequence_num to be always zero?

Reply:
To be clear, you don't increment it inside Enum.reduce, so it is `0` for all statements.

Reply:
This isn't clear to me from the documentation. All the [documentation says](https://dev.mysql.com/doc/internals/en/sequence-id.html) on this is:
> The sequence-id is incremented with each packet and may wrap around. It starts at 0 and is reset to 0 when a new command begins in the Command Phase. 

If this means each "MySQL packet" then it should increase. If it means TCP packet it should not.

Reply:
I will try to have a look at drivers in other languages.

Reply:
sequence_id is for mysql packets, we always start with 0, increase it on every packet we send, and at 255 we go back to 0. We only have 1 byte to store sequence_id in the packet so that's where wrap around comes in.

Each command (com_stmt_prepare, com_stmt_execute etc) is independent - so sequence_id for each of them starts at 0. We'd only increment sequence if we send multiple packets as part of the same command - in practice this only happens if we have a huge payload that doesn't fit in one mysql packet, then we send multiple.

Reply:
I removed the variable and set it to 0